server/register.js

```
@@ -19,5 +21,95 @@ module.exports = ({ strapi }) => {
+    
+    if (adminContentTypeConfigModel) {
+      // Add a lifecycle hook to the admin content type configuration model
+      strapi.db.lifecycles.subscribe({
+        models: ['admin::content-type-configuration'],
+        
+        async beforeUpdate(event) {
+          try {
+            const { params, data } = event;
+            
+            // Check if we're updating plugin settings
+            if (data && data.settings && data.settings.pluginOptions) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)